superset/assets/src/dashboard/util/getPublishSubscriberMap.js

```diff
@@ -119,10 +119,26 @@ function getSubscriberMap(slices, publishers) {
 }
 
 export default function getPublishSubscriberMap(slices) {
+    slices.forEach(slice => {
+        let linkedSlices = getLinkedSlicesFromSubscriberLayer(slice.formData.subscriber_layers);
+        slice.formData.linked_slice = linkedSlices ? linkedSlices : [];
+        slice.formData.actions = ['APPLY_FILTER'];
+    });
     var publishers = getPublisherMap(slices);
```

Comment:
can we move whole this functionality to separate function as updateSlices

Reply:
done